src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function(a, b) {
+      return a + b;
+    },
+
+    subtract: function(a, b) {
+      return a - b;
+    },
+
+    multiply: function(a, b) {
+      return a * b;
+    },
+
+    divide: function(a, b) {
+      return a / b;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(operation, secondValue) {
+      this.result = operation(this.result, secondValue);
```

Comment:
The operate method should be replaced with individual methods for add, subtract, multiply, and divide that update the result property directly. 🔄